### PR TITLE
Publish two packages

### PR DESCRIPTION
**Pull request by JonathanTripp**

### CODE REVIEW

File: hi-ml-azure/README.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+# Microsoft Health Intelligence Machine Learning Toolbox
```
Comment (mebristo) [inline hi-ml-azure/README.md:1]: Should we add "Azure" here? Also on line 9 "ML components will be added in the next few weeks" can be deleted

File: hi-ml-azure/README.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+This toolbox aims at providing low-level and high-level building blocks for Machine Learning / AI researchers and
+practitioners. It helps to simplify and streamline work on deep learning models for healthcare and life sciences,
+by providing tested components (data loaders, pre-processing), deep learning models, and cloud integration tools.
+
+This toolbox is still in very early stages, and presently offers only the cloud integration components. ML components
+will be added in the next few weeks.
+
+## Getting started
+
+* Install from `pypi` via `pip`, by running `pip install hi-ml`
```
Comment (mebristo) [inline hi-ml-azure/README.md:14]: pip install hi-ml-azure

File: hi-ml-azure/README.md (hunk 1 above)
Comment (mebristo) [reply hi-ml-azure/README.md:1]: Line 20 also refers to this as the hi-ml package:  are we going to refer to this separately as this hi-ml-azure package?

**Comment by javier-alvarez** [comment]
Is it possible to have a different folder structure?

E.g. src/health why not just /health/

testhiml/health/azure is also not very readable why not just tests/azure

File: README.md [hunk 3]
```diff
@@ -70,12 +74,27 @@ if __name__ == '__main__':
...
-
-For details, please refer to the [onboarding page](docs/source/first_steps.md).
+on the commandline, like `python myscript.py --azureml`
  
+In the added code, you need to supply the following parameters:
+* `entry_script`: The script that should be run
+* `snapshot_root_directory`: The directory that contains all code that should be packaged and sent to AzureML. All
+Python code that the script uses must be copied over.
+* `compute_cluster_name`: The name of the AzureML cluster that should run the job. This can be a cluster with CPU or 
+GPU machines. See [here for documentation](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-create-attach-compute-studio#amlcompute)
```
Comment (ant0nsc) [inline README.md:84]: I had removed that to make the readme more concise, I'd rather have the messy bits in the docu. Did you work off the latest main?

File: README.md [hunk 4]
```diff
@@ -35,22 +34,27 @@ if __name__ == '__main__':
...
+                                         snapshot_root_directory=current_file.parent,
+                                         workspace_config_path=Path("config.json"),
+                                         compute_cluster_name="preprocess-ds12",
+                                         conda_environment_file=Path("environment.yml"),
                                          input_datasets=["images123"],
                                          # Omit this line if you don't create an output dataset (for example, in
                                          # model training scripts)
-                                         output_datasets=["images123_resized"],
-                                         default_datastore="my_datastore")
+                                         output_datasets=["images123_resized"],)
```
Comment (ant0nsc) [inline README.md:57]: I cleaned that up to reduce clutter in a previous PR, Is there an important reason why you put that back in?

File: docs/source/examples/2/environment.yml [hunk 5]
```diff
@@ -3,4 +3,4 @@ dependencies:
   - pip=20.1.1
   - python=3.7.3
   - pip:
-    - hi-ml==0.1.3
+    - hi-ml-azure==0.1.3
```
Comment (ant0nsc) [inline docs/source/examples/2/environment.yml:6]: There will soon no longer be 0.1.3, can we make all example environments use >=0.1?

**Review by ant0nsc — APPROVED** [review]

**Review by ant0nsc — APPROVED** [review]

**Pull request merged**